src/content/snapshotOptions/threshold.mdx

```diff
@@ -7,88 +7,179 @@ sidebar: { order: 10, label: Threshold for changes }
+        chromatic: { diffIncludeAntiAliasing: true },
+      },
+    };
+
+    export default meta;
+    type Story = StoryObj<typeof UserAccount>;
+    
+    export const Default: Story = {
+      play: async ({ canvasElement }) => {
+        const canvas = within(canvasElement);
+        await expect(canvas.getByText("Welcome, username")).toBeInTheDocument();
+      },
+    };
+    ```
+    <ParamsCallout name="diffIncludeAntiAliasing" integration="storybook" />
+  </Fragment>
+  <Fragment slot="playwright">
+    ```ts title="tests/UserProfile.test.js|ts"
+    import { test, expect } from "@chromatic-com/playwright";
+
+    test.describe("User Profile Page", () => {
+      // Configures this to include anti-aliasing in the diff calculation.
+      test.use({ diffIncludeAntiAliasing: true });
+
+      test("Successfully loads the user information ", async ({ page }) => {
+        await page.goto("/profile");
+        await expect(page.getByText("Welcome, username")).toBeVisible();
+      });
+    });
+    ```
+    <ParamsCallout name="diffIncludeAntiAliasing" integration="playwright" />
+  </Fragment>
+  <Fragment slot="cypress">
+    ```ts title="cypress/e2e/UserProfile.cy.js|ts"
+    describe("User Profile Page", () => {
+      // Configures this to include anti-aliasing in the diff calculation.
+      it("Successfully loads the user information", { env: { diffIncludeAntiAliasing: true } }, () => {
+        cy.visit("/profile");
+        cy.get("span").contains("Welcome, username");
+      });
+    });
+    ```
+    <ParamsCallout name="diffIncludeAntiAliasing" integration="cypress" />
+  </Fragment>
+</IntegrationSnippets>
+
+{/* prettier-ignore-end */}
+
+## Troubleshooting
 
 <details>
 <summary>Why weren't diffs found when a color or background changed?</summary>
 
-Chromatic uses a threshold to determine how much needs to visually change between snapshots before they get flagged as changes. This prevents false positives due to anti-aliasing and other non-deterministic rendering artifacts.
-
-But our default threshold may result in subtle changes being missed. For instance, nuanced changes to the shade of gray of a background. In this case, you may want to adjust the `diffThreshold` to be more accurate (lower value).
-
-Find the right threshold for your UI using our [**interactive diff tool**](https://6262c53f521620003ac2ff49-ukmsdlppcb.chromatic.com/?path=/story/stories-diff-threshold-check--test-yours-out).
+Chromatic uses a threshold to determine how much must change between snapshots before they get flagged as visual changes. This prevents false positives due to anti-aliasing and other non-deterministic rendering artifacts.
 
-</details>
+Chromatic uses a threshold to determine how much needs to visually change between snapshots before they get flagged as changes. This prevents false positives due to anti-aliasing and other non-deterministic rendering artifacts.
```

Comment:
Duplicate of above?

Reply:
Thanks for catching that, I removed the duplicate item with the latest commit